scripts/verify-permissions.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2021 Paul Keith <javelinanddart@gmail.com>
+#
```

Comment:
If you made any changes, also add your own name

Reply:
You mean instead of the original author's name or next to it? I've placed my name lower on line 14.

Reply:
IMHO you should move line 14 to here